### ISSUE TITLE
Touchscreen and tablet (windows ink)

### ISSUE THREAD

**Issue by Neil-Sawhney** [CE-Programming/CEmu#397]
(Thanks for reporting an issue! Please make sure to fill out the blanks below.)

What's wrong, and with what software version?
---------------------------------------------
**Operating System:** …  Windows 10
**CEmu version:** … (see in the `About CEmu` menu)  1.3
**Describe your issue:**  
… (including, if applicable what you expect vs. what happens)  

Cannot click any buttons with a touchscreen or using a tablet, sometimes one button will press but then freeze the whole program. This is unfortunate because a touchscreen would be the fastest/easiest way to use the emulator without remembering all the key bindings.

What are the steps to reproduce this issue?
-------------------------------------------
1. … press buttons with a tablet(such as a huion) or touchscreen
2. …
3. …

Any logs, error output, screenshot, other comments...?
------------------------------------------------------
…

**Comment by adriweb** [comment]
Qt doing weird things again maybe....
Might be related to #348 and #282.

Would be worth trying a build with Qt6 in case they improved it (which I think they did)

**Comment by Neil-Sawhney** [comment]
appreciate the quick reply! The download links for the windows development builds seems to be broken? [https://ce-programming.github.io/CEmu/download]

**Comment by jacobly0** [comment]
Yes, due to technical issues that will take a while to sort out, there are no CEmu dev builds available at this time.

**Comment by thespyboss** [comment]
Just letting the devs know, this also happens for me on Linux when trying to use a Wacom Tablet with the emulator. Clicks don't register on the calculator.

**Comment by superbeelavaman** [comment]
This issue also happens when using in the Linux subsystem on a Chromebook that has an integrated touchscreen (Samsung Galaxy Chromebook 2), so I'm guessing it's just overall a problem with how programs interface with touchscreens.

**Comment by adriweb** [comment]
This is actually a duplicate of #348 , closing here since there are more technical details on the other one anyway.
